src/Workspaces/Core/Portable/FindSymbols/SymbolFinder_Helpers.cs

```diff
@@ -54,8 +54,8 @@ private static bool IsAccessible(ISymbol symbol)
                 var namespace2Count = namespace2.ConstituentNamespaces.Length;
                 if (namespace1Count != namespace2Count)
                 {
-                    if ((namespace1Count > 1 && await namespace1.ConstituentNamespaces.AnyAsync(n => NamespaceSymbolsMatchAsync(solution, n, namespace2, cancellationToken)).ConfigureAwait(false)) ||
-                        (namespace2Count > 1 && await namespace2.ConstituentNamespaces.AnyAsync(n2 => NamespaceSymbolsMatchAsync(solution, namespace1, n2, cancellationToken)).ConfigureAwait(false)))
+                    if ((namespace1Count > 1 && await namespace1.ConstituentNamespaces.AnyAsync(static (n, arg) => NamespaceSymbolsMatchAsync(arg.solution, n, arg.namespace2, arg.cancellationToken), (solution, namespace2, cancellationToken)).ConfigureAwait(false)) ||
```

Comment:
Can the other extension method be deleted in favor of the new one?

Reply:
I'm not sure. The other one is perfectly valid so I didn't look outside the ones that appeared in the trace.

Reply:
I gave a look and it seems this PR replaced all usages of the old extension method.

https://grep.app/search?q=.AnyAsync&filter[repo][0]=dotnet/roslyn

It had only 3 usages which you replaced in the PR. I think it should be fine either way.